PRESUBMIT.py

```diff
@@ -0,0 +1,278 @@
+# Copyright 2018 The Chromium Authors. All rights reserved.
+# Use of this source code is governed by a BSD-style license that can be
+# found in the LICENSE file.
```

Comment:
The license-file is Apache2 in this repo.  So... what do we do here to make it right?